backend/attendance/src/main/resources/attendance-service-persister.yml

```
@@ -0,0 +1,408 @@
+
+    - version: 1.0
+      name: Attendance Log
+      description: Persists attendance logs and corresponding documents
+      fromTopic: save-attendance-log
+      isTransaction: true
+      isAuditEnabled: true
+      module: ATND
+      objecIdJsonPath: $.id
+      tenantIdJsonPath: $.tenantId
+      transactionCodeJsonPath: $.registerId
+      auditAttributeBasePath:  $.attendance.*
```

yamllint (1.35.1)

[error] too many spaces after colon

(colons)